Add instructions on how to merge translations (#195)
nextflow-io/training: TRANSLATING.md

2. Work locally (see below) and make your changes
    - Check if the language you want to translate to is already enabled in the `mkdocs.yml` file. If it isn't do it according to [this](https://github.com/nextflow-io/training/pull/163/files#diff-98d0f806abc9af24e6a7c545d3d77e8f9ad57643e27211d7a7b896113e420ed2) example.
    - If you want to improve an already existing translation, the file already exists and the language is already set up. Simply open the file and work on it. Otherwise, create the new file with the following pattern: If the original file in English is `filename.md`, you will create in the same folder a new file named `filename.language_code.md`, where `language_code` is the language code that you can find [here](https://en.wikipedia.org/wiki/ISO_639-1), for the language you wish to translate to. Pay attention to the language code, as it has to be the same that is specified in the `mkdocs.yml` file for that language.

>     - Ideally, for new files, copy-paste the original English file with the new filename and commit. This way, it's easier to review your pull request by seeing the original file in English vs the translated changes you did.

Reviewer: ```suggestion
    - Ideally, for new files, copy-paste the original English contents into the new file and commit _before_ starting to translate. This way, it's easier to review your pull request by seeing the original file in English vs the translated changes you did.
```

-   [Portuguese](https://docs.google.com/spreadsheets/d/1HUa3BO2kwukhX4EXQ-1blXeP5iueUdM23OwDRpfarDg/edit?usp=sharing)

## Merging

> Aiming at a more comprehensive git history, pull requests will be squashed with a commit message outlining:

Author: Great idea!

  - What file(s)/page(s) the PR translated (preferably, 1 PR per file)
  - What training material the translated file(s)/page(s) are part of
  - What language it was translated into

This practice gives more freedom/space for committers to get into more detail in their commit title/message without having to add the info above. For example:

```

> PR: Translate to Brazilian Portuguese the Tower section of the basic training

Author: I thought of making it a code block to make it explicit that this would go in the git commit message, but maybe it's confusing. I will do as you suggested 😄

> PR: Translate to Brazilian Portuguese the Tower section of the basic training

Author: Ops, my bad. I thought of this for the second block but did to both 🙄 😆
